.github/workflows/update_gbs_cache.yml

```diff
@@ -98,3 +110,26 @@ jobs:
         pushd nntrainer
         gbs build --skip-srcrpm -A ${{ matrix.gbs_build_arch }} ${{ matrix.gbs_build_option }} --define "_skip_debug_rpm 1"
         popd
+
+  print_result:
+    runs-on: ubuntu-22.04
+    needs: [cache_gbs_build]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v1
+
+      - name: print-result
+        run: |
+          mkdir -p ~/testresult
+          pip install pybadges setuptools
+          echo '${{ toJSON( needs.cache_gbs_build.outputs) }}' > gbs_build_result.json
+          python3 .github/workflows/gen_gbs_result_badge.py gbs_build_result.json ~/testresult
+          git clone https://${{ secrets.TAOS_ACCOUNT }}:${{ secrets.TAOS_ACCOUNT_TOKEN }}@github.com/nnstreamer/nnstreamer.github.io.git
+          pushd nnstreamer.github.io
+          cp -r ~/testresult/*.svg testresult
+          git config user.email "nnsuite@samsung.com"
+          git config user.name "nnsuite"
+          git add *
+          git commit -s -m "Update unit test result badge."
+          git push origin main -f
```

Comment:
By the way, regarding this part,
I think it could be reused similarly in multiple workflows.
it would be possible to reuse the code if you use 'composite action'. Thank you.

https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action

Reply:
Badges and logs should be uploaded for other platforms(android, yocto and etc.).
It would be nice to use composite action or reusable workflow to reduce the duplicated codes.
I will work it in the next PR.

Reply:
Thank you, have a nice holiday. :+1: